ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -226,6 +226,7 @@ descriptor:
   sweeper: "Action performed in a sump to prevent sediment build-up via water turbulence."
   sync: "The simultaneous operation or activity of two or more things."
   tertiary: "Associated with the tertiary (peripheral) loops of a production process."
+  transducer: "A transducer is a device that coverts high frequency signals into high frequency sound waves (e.g. used to repel rodent)."
```

Comment:
make more generic: "A device that converts energy from one form to another (e.g., sound energy to electrical energy in a microphone, electrical energy to sound energy in a speaker, etc.)."

Reply:
Ohkay . Changes done